zedd-app/src/AppState.ts

```diff
@@ -739,6 +747,7 @@ export class AppState {
     } else {
       this.markedSlices.push(slice)
       this.slicesMarked = true
+      this.sliceToCopy = this.markedSlices.pop()
```

Comment:
Hast du überprüft, dass das Array `this.markedSlices` nicht verändert wird? Ich denke nämlich, dass `pop` das tut.